### PR TITLE
Add Bluetooth CGMS service UUID

### PR DESCRIPTION
**Pull request by wentong-li**
Add Bluetooth continuous glucose monitoring service (CGMS)-related uuid, including 1 service uuid and 7 characteristic uuid.
- Add BT_UUID_CGMS for Continuous Glucose Monitoring
- Add BT_UUID_CGM_MEASUREMENT for CGM Measurement
- Add BT_UUID_CGM_FEATURE for CGM Feature
- Add BT_UUID_CGM_STATUS for CGM Status
- Add BT_UUID_CGM_SESSION_START_TIME for CGM Session Start Time
- Add BT_UUID_CGM_SESSION_RUN_TIME for CGM Session Run Time
- Add BT_UUID_CGM_SPECIFIC_OPS_CONTROL_POINT for CGM Specific Ops Control Point
- Add BT_UUID_RECORD_ACCESS_CONTROL_POINT for Record Access Control Point

### CODE REVIEW

**Comment by carlescufi** [comment]
@wentong-li please format your commit message as per the guidelines here:
https://docs.zephyrproject.org/latest/contribute/index.html#commit-guidelines

**Comment by wentong-li** [comment]
The commit messages have been updated.

File: include/bluetooth/uuid.h [hunk 1]
```diff
@@ -2123,6 +2123,78 @@ struct bt_uuid_128 {
...
+ */
+#define BT_UUID_CGMS \
+	BT_UUID_DECLARE_16(BT_UUID_CGMS_VAL)
+/** @def BT_UUID_CGM_MEASUREMENT_VAL
+ *  @brief CGM Measurement Characteristic value
+ */
+#define BT_UUID_CGM_MEASUREMENT_VAL 0x2aa7
+/** @def BT_UUID_CGM_MEASUREMENT
+ *  @brief CGM Measurement Characteristic
+ */
```
Comment (Thalley) [inline include/bluetooth/uuid.h:2141]: We have so far ordered our UUID entried by increasing UUID value. Please insert these in the correct places based on the values.

**Review by Thalley — CHANGES_REQUESTED** [review]

File: include/bluetooth/uuid.h (hunk 1 above)
Comment (wentong-li) [reply include/bluetooth/uuid.h]: Updated. They are in ascending order now.

**Review by jhedberg — DISMISSED** [review]

**Comment by wentong-li** [comment]
updated the author info to meet the format according to the error message

**Review by Thalley — DISMISSED** [review]

**Review by jhedberg — DISMISSED** [review]

**Comment by wentong-li** [comment]
Updated the commit message body to meet the length limit.

**Review by Thalley — DISMISSED** [review]

**Comment by wentong-li** [comment]
Can anyone help? Please let me know if further modification is required. Thanks!

**Comment by Thalley** [comment]
@wentong-li Please rebase the PR :)

**Comment by lairdjm** [comment]
@wentong-li I think something has gone a bit wrong here...

**Comment by wentong-li** [comment]
> @wentong-li I think something has gone a bit wrong here...

Sorry for wrong operation. Just fixed...

**Review by Thalley — APPROVED** [review]

**Review by cfriedt — APPROVED** [review]

**Review by jhedberg — APPROVED** [review]

**Pull request merged**